src/mono/mono/mini/aot-compiler.c

```diff
@@ -14896,8 +14896,7 @@ aot_assembly (MonoAssembly *ass, guint32 jit_opts, MonoAotOptions *aot_options)
 	acfg->stats.jit_time = GINT64_TO_INT (TV_ELAPSED (atv, btv));
 	// Current implementation uses dedup_methods hash table for storing extra methods which are emitted in a dedup AOT image.
 	// Previously, cfg->skip flag in dedup_skip_methods is used for indicating if a method should be emitted in an AOT image.
-	// Method dedup_skip_methods is used only for wasm.
-#ifdef TARGET_WASM
+#if 0
```

Comment:
I suppose the guarded code can be removed all together

Reply:
Correct, wasm uses llvm only, so `dedup_skip_methods` hasn't been used. There is a `cfg->skip` that could be removed as an obsolete flag. Do you think it is a good idea to remove it in this PR?

Reply:
Would suggest removing that code in a separate PR.